New branch containing completed tasks

mgandore opened the pull request.

mgandore committed:
Add tasks

mgandore committed:
fix indentation

mgandore committed:
Improve code readability

mgandore committed:
Add required indentation

danielursu commented on tasks-june/script.js:
```diff
@@ -0,0 +1,112 @@
+	let allOptions = document.querySelectorAll('option');
+	for (let item of allOptions) {
+		if (toDelete === item.value) {
+			item.remove();
```
remove from array

danielursu commented on tasks-june/script.js:
```diff
@@ -0,0 +1,112 @@
+function deleteOp() {
+	let toDelete = dropdown.value;
+	let allOptions = document.querySelectorAll('option');
+	for (let item of allOptions) {
```
you can use filter on array
for example
array = [1,2,3,4]
value = 3
result = array.filter((item) => item !== value); // result = [1,2,4]

danielursu commented on tasks-june/script.js:
```diff
@@ -0,0 +1,112 @@
+	raza unei sfere. Sa se afiseze volumul corespunzator razei.
+*/
+function getVolume() {
+	console.log(radius.value);
```
remove console.log

danielursu commented on tasks-june/script.js:
```diff
@@ -0,0 +1,112 @@
+	boldWords = document.getElementsByTagName('b');
+}
+
+function hightlight() {
+	for (let word of boldWords) {
+		word.style.backgroundColor = 'orange';
+		word.style.fontWeight = 100;
+	}
+}
+
+function reset() {
+	for (let word of boldWords) {
+		word.style.backgroundColor = 'white';
+		word.style.fontWeight = 700;
+	}
+}
```
you can improve a little bit. Did you notice the duplicate code?
```suggestion
function setStyleOnBoldWords(backgroundColor, fontWeight) {
        for (let word of boldWords) {
	        word.style.backgroundColor = backgroundColor;
		word.style.fontWeight = fontWeight;
	}
}

function hightlight() {
	setStyleOnBoldWords('orange', 100);
}

function reset() {
	setStyleOnBoldWords('white', 700);
}
```

danielursu replied:
in this case, you only modify the parameters

mgandore committed:
Change approach on Exercice 2

Insted of creating new variables (allOptions, toDelete) which
contained a list of nodes and parsing it 'manually' to find the proper
node to delete I decided to take advantage of the <select> element's
method .remove(index) and property .selectedIndex.

*Since there was no straightforward way to benefit from the array
method,filter, because the only solution that I came up with
supposed to: use the initial NodeList to create a new array then filter
it, then create a new Node list and replace it, i gave up on that.

mgandore committed:
Delete comment
